amethyst_renderer/src/pass/flat/interleaved.rs

```diff
@@ -140,8 +145,14 @@ where
             ..
         } = self;
 
-        // TODO: multiple cameras
-        let camera = (&camera, &global).join().next();
+        let camera: Option<(&Camera, &Transform)> = active
+            .and_then(|a| {
+                let cam = camera.get(a.entity);
+                let transform = global.get(a.entity);
+                cam.into_iter().zip(transform.into_iter()).next()
```

Comment:
This is unusual but good alternative to `cam.and_then(|cam| transform.map(|transform| (cam, transform)))` 😄

Reply:
That one doesn't work because of borrow rules.